### DISCUSSION TITLE
Local dev start

### DISCUSSION THREAD

**Discussion by blondie63** [devaslanphp/help-desk#135] [Q&A]
I managed to start the application locally on my Mac without using Docker but I always have to run "php artisan serve" on one terminal session and "npm run dev" on another terminal session, is it possible to avoid this double session?

**Comment by heloufir** [comment]
Hello,

I think there is no other solution to run vite (in dev mode) and serve the application in the same command line.

But you can execute: npm run build
To build assets then you can run: php artisan serve
But you need to run the build command on every change you made on your views and styles/scripts.

Please use the issue section to create issues and help requests.
